pom.xml

```diff
@@ -47,10 +47,10 @@
     <java.require.version>11</java.require.version>
     <jdeps.multiRelease>11</jdeps.multiRelease>
     <junit.version>4.13.2</junit.version>
-    <kotlin.version>1.7.20</kotlin.version>
+    <kotlin.version>1.8.10</kotlin.version>
     <maven-dependency-plugin.version>3.3.0</maven-dependency-plugin.version>
-    <maven-plugin-tools.version>3.6.4</maven-plugin-tools.version>
-    <maven-resolver.version>1.8.2</maven-resolver.version>
+    <maven-plugin-tools.version>3.7.1</maven-plugin-tools.version>
```

Comment:
Quite a bit of time passed since this PR was created and, hence, this should probably be bumped again (3.15.2 atm).

Issues:
- The deprecated `PluginUtils.MAVEN_COMPONENTS` is gone which causes a compilation failure in `KotlinMojoDescriptorExtractor.kt:305`. Note how the deprecation warning was suppressed!
- At `SourceScanner.kt:122/123` the two references in `PluginToolsRequest` don't exist anymore.